x/claim/genesis_test.go

```diff
@@ -20,9 +20,9 @@ var testGenesis = types.GenesisState{
 	ModuleAccountBalance: sdk.NewInt64Coin(types.DefaultClaimDenom, 750000000),
 	Params: types.Params{
 		AirdropStartTime:   now,
-		DurationUntilDecay: types.DefaultDurationUntilDecay, // 2 month
-		DurationOfDecay:    types.DefaultDurationOfDecay,    // 4 months
-		ClaimDenom:         types.DefaultClaimDenom,         // uosmo
+		DurationUntilDecay: types.DefaultDurationOfDecay,
+		DurationOfDecay:    types.DefaultDurationOfDecay,
+		ClaimDenom:         types.DefaultClaimDenom, // uosmo
```

Comment:
These changes look like a bug?

Reply:
hmm i dont know why but the changes in github are not showing correctly they are changed as well when you look at the file changed!